lms/envs/devstack.py

```diff
@@ -435,7 +438,3 @@ def should_show_debug_toolbar(request):
 # ]
 # TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
 # derive_settings(__name__)
-
-# Uncomment the lines below if you'd like to see SQL statements in your devstack LMS log.
```

Comment:
I think we will want to add these back in.

Reply:
Not sure how they got deleted 🤔